Code/MinimalLib/cffiwrapper.cpp

```diff
@@ -62,7 +62,7 @@ using namespace RDKit;
 #endif
 
 namespace {
-char *str_to_c(const std::string &str, size_t *len = nullptr) {
+char *str_to_c(const std::string &str, size_t *len = NULL) {
```

Comment:
Why is this change being made? This is C++ code, so `nullptr` should still be fine, right?

Reply:
same comment applies to all the other `NULL` replacements in this file